rhobs/alerting/data_plane/prometheus.release_service_alerts.yaml

```diff
@@ -59,3 +59,27 @@ spec:
         alert_team_handle: <!subteam^S03SVBS426R>
         runbook_url: https://gitlab.cee.redhat.com/konflux/docs/sop/-/blob/main/release-service/release-processing-latency.md?ref_type=heads
         team: release
+
+    - alert: ReleaseServiceControllerManagerAvailability
+      expr: |
+        (
+          avg_over_time
+          (
+            (sum(kube_pod_container_status_ready{namespace="release-service", container="manager"}) by(namespace))
+            [1h:]
```

Comment:
Is it intentional that we have 1h at the numerator and 1d below?

Reply:
Yes. The reason is that we might one of the metrics absent for a moment (maybe in the hour it checked it and I the calculation would be done with a missing node. Using one day (theoretically) I can have a  "good enough" assumption of how many nodes I should have up in that hour. I am open to suggestion.

Reply:
I think we should have a test case for that scenario.

Reply:
added.

Reply:
Cool. Can you also add a test case in which the alert is not firing?

Reply:
added.